scripts/docker/buildpack-deps/Dockerfile.ubuntu1604.clang.ossfuzz

```diff
@@ -22,7 +22,7 @@
 # (c) 2016-2019 solidity contributors.
```

Comment:
```suggestion
# (c) 2016-2021 solidity contributors.
```
Maybe it would be better to get rid of this comment though? Seems pointless to me.